docker/onboard/Dockerfile

```diff
@@ -58,6 +59,9 @@ RUN cd torch_deps && \
 # Install Detecto
 RUN pip3 install --no-cache-dir detecto
 
+# Install DepthAI
+RUN pip3 install depthai
```

Comment:
Can this be added to a previous pip install command (either the one that has saleae at the end or the detecto one)? Otherwise, add --no-cache-dir to reduce image size.